integration/internal/api/image/image.go

```diff
@@ -0,0 +1,66 @@
+package image
+
+import (
+	"context"
+	"io"
+	"os"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+)
+
+// Save saves an image to a tarball names by path
+func Save(client client.APIClient, path, image string) error {
```

Comment:
Can be replaced with a `saveToPath(source io.Reader, path string) err`

Reply:
It's not clear to me where `saveToPath` is implemented or how it can be a replacement with such a different signature.